src/watson/registration.py

```diff
@@ -151,6 +152,18 @@ def get_meta(self, obj):
             for field_name in self.store
         )
 
+    def serialized_meta(self, obj):
```

Comment:
**Final** pedantic note: this should be `seialize_meta`, for consistency with the other method. I think a typo here. :)

Fix this, and I'll happily pull. This is a great addition to the project!